react-scanner.config.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+	// Crawl the entire repo
+	crawlFrom: './',
+	// Needed for properly reporting components with dot notation
+	includeSubComponents: true,
+	// Exclude usage in tests, stories, and React Native files.
```

Comment:
I ran the command on my machine and it looks like the report is including some React Native usage information. I don't think that's expected?

![Screenshot 2024-10-23 at 23 51 48](https://github.com/user-attachments/assets/3c499e44-8984-45ec-9e09-8363c09d0f6f)

Reply:
Ah, good catch. Seems like the `globs` doesn't work as expected. Seems like it uses another tool for the globbing part with different syntax - I'll fine-tune it next week.

Reply:
This should be fixed in https://github.com/WordPress/gutenberg/pull/65463/commits/a39d4dbd4ddfe95627d131a1cb37c248f0233178.